packages/block-editor/src/utils/test/transform-styles.js

```diff
@@ -4,6 +4,28 @@
 import transformStyles from '../transform-styles';
 
 describe( 'transformStyles', () => {
+	it( 'should not throw error in case of invalid css', () => {
```

Comment:
This test is welcome but it actually doesn't not test what was broken in the UI of the custom CSS advanced panel. It just tests the return value is `null` when there is a CSS error. It doesn't make sure the UI works as expected.